nosave-1.ks.in

```diff
@@ -0,0 +1,22 @@
+#test name: nosave-1
+url @KSTEST_URL@
+install
+network --bootproto=dhcp
+
+bootloader --timeout=1
+zerombr
+clearpart --all --initlabel
+autopart
+
+keyboard us
+lang en
+timezone America/New_York
+rootpw qweqwe
+shutdown
+
+%packages
+%end
+
+%post
+echo SUCCESS > /root/RESULT
+%end
```

Comment:
GitHub is complaining on missing newline character at the end of the last line in `*.ks.in` files. Could you please make the GitHub happy :)